src/main/java/org/broadinstitute/hellbender/utils/recalibration/BaseRecalibrationEngine.java

```diff
@@ -337,7 +337,7 @@ private GATKRead setDefaultBaseQualities( final GATKRead read ) {
                 // knownSite is outside clipping window for the read, ignore
                 continue;
             }
-            int featureStartOnRead = ReadUtils.getReadCoordinateForReferenceCoordinate(softStart, cigar, knownSite.getStart(), ReadUtils.ClippingTail.LEFT_TAIL, true);
+            int featureStartOnRead = ReadUtils.getReadCoordinateForReferenceCoordinate(softStart, cigar, knownSite.getStart(), ReadUtils.ClippingTail.RIGHT_TAIL, true);
```

Comment:
Can you add a unit or integration test that demonstrates the problem being fixed (ie., a test that fails without this fix and passes with it)

Reply:
added a unit test